<launch>

    <arg name="portDevA" default="0" />
    <arg name="portDevB" default="1" />

    <node pkg="hardware_abstraction" type="mainProgramTask2.py" name="mainProgram" output="screen" />    

    <node pkg="rosserial_python" type="serial_node.py" name="deviceA" output="screen">
        <param name="port" value="/dev/ttyACM$(arg portDevA)" />
        <param name="baud" value="9600" />
    </node>

    <node pkg="rosserial_python" type="serial_node.py" name="deviceB" output="screen">
        <param name="port" value="/dev/ttyACM$(arg portDevB)" />
        <param name="baud" value="9600" />
    </node> 

</launch>